docs/application/toc_application.md

```diff
@@ -783,9 +783,9 @@
 ##### [Tracepoints](/application/native/guides/performance/tracepoints.md)
 
 #### Watchface Complication
-##### [Overview](/application/native/guides/compliations/overview.md)
-##### [Watchface Complication](/application/native/guides/compliations/watchface-complication.md)
-##### [Complication Provider](/application/native/guides/compliations/watchface-complication-provider.md)
+##### [Overview](/application/native/guides/complications/overview.md)
```

Comment:
toc_application.md is a backup file, so you don't need update it.
This file is not in use now.